parquet-cli/src/main/java/org/apache/parquet/cli/Util.java

```diff
@@ -151,6 +151,8 @@ public static String shortCodec(CompressionCodecName codec) {
         return "B";
       case LZ4:
         return "4";
+      case LZ4_RAW:
+        return "4";
```

Comment:
this looks like the same value as LZ4, which doesn't seem correct, as they are in fact two different compression schemes?

Reply:
Change the short name to "F".